examples/tutorials/api/estimators.py

```diff
@@ -0,0 +1,286 @@
+
+   -   **Model Fitting (Forward Folding)**: Refit the flux of a model component
+       within specified energy, time, or spatial regions.
+   -   **Excess Calculation (Backward Folding)**: Use the analytical solution by Li and Ma
+       for significance based on excess counts, currently available in `~gammapy.estimators.ExcessMapEstimator`.
+
+For further information on these details please refer to :doc:`/user-guide/estimators`.
+
+The setup
+---------
+
+"""
+
+import numpy as np
+import matplotlib.pyplot as plt
+from astropy import units as u
+from IPython.display import display
+from gammapy.datasets import SpectrumDatasetOnOff, Datasets, MapDataset
+from gammapy.estimators import (
+    FluxPointsEstimator,
+    ExcessMapEstimator,
+    FluxPoints,
+)
+from gammapy.modeling import Fit
+from gammapy.modeling.models import SkyModel, PowerLawSpectralModel
+from gammapy.utils.scripts import make_path
+
+
+######################################################################
+# Flux Points Estimation
+# ----------------------
+#
+# We start with a simple example for flux points estimation taking multiple datasets into account.
+# In this section we show the steps to estimate the flux points.
+# First we read the pre-computed datasets from `$GAMMAPY_DATA`.
+#
+
+datasets = Datasets()
+path = make_path("$GAMMAPY_DATA/joint-crab/spectra/hess/")
+
+for filename in path.glob("pha_obs*.fits"):
+    dataset = SpectrumDatasetOnOff.read(filename)
+    datasets.append(dataset)
+
+######################################################################
+# Next we define a spectral model and set it on the datasets:
+#
+
+pwl = PowerLawSpectralModel(index=2.7, amplitude="5e-11  cm-2 s-1 TeV-1")
+datasets.models = SkyModel(spectral_model=pwl, name="crab")
+
+######################################################################
+# Before using the estimators, it is necessary to first ensure that the model is properly
+# fitted. This applies to all scenarios, including light curve estimation. To optimize the
+# model parameters to best fit the data we utilise the following:
+#
+
+fit = Fit()
+fit_result = fit.optimize(datasets=datasets)
+print(fit_result)
+
+######################################################################
+# A fully configured Flux Points Estimation
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+#
+# The `~gammapy.estimators.FluxPointsEstimator` estimates flux points for a given list of datasets,
+# energies and spectral model. The most simple way to call the estimator is by defining both
+# the ``source`` and ``energy_edges``.
+# Here we prepare a full configuration of the flux point estimation.
+# Firstly we define the ``backend`` for the fit:
+#
+
+fit = Fit(
+    optimize_opts={"backend": "minuit"},
+    confidence_opts={"backend": "scipy"},
+)
+
+
+######################################################################
+# Define the fully configured flux points estimator:
+#
+
+energy_edges = np.geomspace(0.7, 100, 9) * u.TeV
```

Comment:
Why geomspace?

Reply:
Isn't it better to show how to downsample the original energy axis so that the edges can be perfectly aligned?

Reply:
Instead I added to the note to ensure users know how this could be done. 
Given the low statistics at lower energies for this example, it is easier to define an energy binning, otherwise we would need to downsample and then select only the higher energies (this looks quite messy)